### DISCUSSION TITLE
What's the name of the font?

### DISCUSSION THREAD

**Discussion by ShahriarKh** [tabler/tabler-icons#270] [Q&A] [answered]
What's the name of the font used in icons?
![image](https://user-images.githubusercontent.com/31452340/180417284-1d1a9aaa-a0ae-48da-b71d-2c077ea2136f.png)
![image](https://user-images.githubusercontent.com/31452340/180417375-168121fc-9099-4aa1-8952-37d55188063b.png)

**Comment by codecalm** [comment, marked as answer]
it's font made by myself :)

**Reply by amytimed** [reply]
Can we get the font? it is cool

**Reply by amytimed** [reply]
If we could get a font of it, we could easily get new text for stuff without icon requests every time

**Comment by ShahriarKh** [comment]
See https://github.com/tabler/tabler-icons/releases/tag/v2.38.0
